src/mantine-demos/src/demos/hooks/use-document-visibility/usage.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react';
+import { Group, Badge } from '@mantine/core';
+import { useDocumentVisibility } from '@mantine/hooks';
+
+const code = `
+import React, { useEffect } from 'react';
+import { Group, Badge } from '@mantine/core';
+import { useDocumentVisibility } from '@mantine/hooks';
+
+  useEffect(() => {
+    console.log(documentVisibility);
+  }, [documentVisibility]);
+
+  return (
+    <Group position="center">
+      <Badge color="blue">Document Visibility: {documentVisibility}</Badge>
+    </Group>
+  );
+}`;
+
+function Demo() {
+  const documentVisibility = useDocumentVisibility();
+
+  useEffect(() => {
+    // eslint-disable-next-line no-console
+    console.log(documentVisibility);
```

Comment:
Do not use console.log in demos, instead change page title with use-document-title hook